modules/iceoryx2/0.5.0/MODULE.bazel

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache Software License 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+# which is available at https://opensource.org/licenses/MIT.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+module(
+    name = "iceoryx2",
+    version = "0.5.0",
+    bazel_compatibility = [">=7.2.1"],
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "googletest", version = "1.15.2")
+bazel_dep(name = "iceoryx", version = "2.95.3")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "rules_cc", version = "0.1.0")
+bazel_dep(name = "rules_rust", version = "0.56.0")
+
+rust = use_extension("@rules_rust//rust:extensions.bzl", "rust")
+rust.toolchain(
+    edition = "2021",
+)
+use_repo(rust, "rust_toolchains")
+
+register_toolchains("@rust_toolchains//:all")
+
+crate = use_extension("@rules_rust//crate_universe:extensions.bzl", "crate")
+crate.from_cargo(
+    name = "crate_index",
```

Comment:
I know too little about rules_rust to determine whether this is safe - what happens if some other module also specifies this name?

Reply:
I'm sorry, but I'm not sure. @dep_module//:crate_index and @root_module//:crate_index are confusing for bazel?
Is it safer to change name like `iceoryx2_crate_index`?

Reply:
That depends on whether these libraries may end up getting linked together with user-supplied code. If so, versions need to align and you need to share a dependency closure. If not, a separate namespace is probably more efficient.

I don't know this extension or your use case well enough to make a call. I would suggest asking for advice in the #rust slack channel.